include/hardware.h

```diff
@@ -45,7 +46,9 @@ bool SB_Get_Address(uint16_t &sbaddr, uint8_t &sbirq, uint8_t &sbdma);
 bool TS_Get_Address(Bitu& tsaddr, Bitu& tsirq, Bitu& tsdma);
 
 extern uint8_t adlib_commandreg;
-FILE * OpenCaptureFile(const char * type,const char * ext);
+
+std::string CAPTURE_GetScreenshotFilename(const char *type, const char *ext);
+FILE *OpenCaptureFile(const char *type, const char *ext);
 
```

Comment:
This is used in `opl.cpp` and `serialport.cpp` too. Should the `CAPTURE_` prefix be added?

Reply:
Great catch - thanks!
Cleaned up.